openmc/lib/core.py

```diff
@@ -415,6 +421,44 @@ def run(output=True):
         _dll.openmc_run()
 
 
+def sample_external_source(n_samples=1, prn_seed=None):
+    """Sample external source
+
+    .. versionadded:: 0.13.1
+
+    Parameters
+    ----------
+    n_samples : int
+        Number of samples
+    prn_seed : int
+        PRNG seed; if None, one will be generated randomly
```

Comment:
```suggestion
        Pseudorandom number generator (PRNG) seed; if None, one will be generated randomly
```
I wonder if it is worth putting  pseudorandom number generator (PRNG) for users not familiar with the acronym